src/gosling-genomic-axis/axis-track.ts

```diff
@@ -566,6 +574,7 @@ function AxisTrack(HGC: any, ...args: any[]): any {
                 this.allTexts.push({
                     importance: chrText.hashValue,
                     text: chrText,
+                    // @ts-expect-error possibly null or undefined and should just be SimpleRope
```

Comment:
Confusing logic here. `rope` can be `SimpleRope | null | undefined` and was assumed to be `SimpleRope` since `rope` was `any` previously.